SpatialGDK/Extras/schema/heartbeat.schema

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+package unreal;
+
+type HeartbeatEvent {
+}
+
+component Heartbeat {
+    id = 100007;
```

Comment:
Oh shoot I'm using this for the startup actor manager. Could it be 100008?

Reply:
Chris is using 100008 for GSM control message, please use 100009.